Modules/MeasureMixing.jl

```diff
@@ -0,0 +1,54 @@
+#=
+    Diagnostics to measure mixing.
+    These are:
+     - variance of concentration over the grid
+     - evolution of the second moment of the tracer patch.
+    
+=#
+module MeasureMixing
+
+export
+    conc_var!
+    area_tracer_patch
+
+using Distributions, GeophysicalFlows
+
+#Calculate the variance of the tracer concentration.
+function conc_var!(conc_var, prob) 
+
+    nlayers = prob.params.nlayers
+    step = prob.clock.step + 1
+    temp = zeros(nlayers)
+    for i in 1:nlayers
+        temp[i] = var(prob.vars.c[:, :, i])
+    end
+    @. conc_var[step, :] = temp
```

Comment:
would this be clearer? (and also avoid allocating `temp`?)

```suggestion
    for i in 1:nlayers
        @. conc_var[step, i] = var(prob.vars.c[:, :, i])
    end
```

Reply:
Yes this is much clearer!